INDEXING.md

```diff
@@ -47,15 +47,15 @@ The simplest usage is to instantiate `PidGatherer` and just call either `pid_lis
-`Argo::IndexingJob` is a subclass of `ActiveJob::Base`.  Its `perform` method just calls `Argo::Indexer.reindex_pid_list` (or its profiled counterpart, based on `Settings.INDEXING_JOB.SHOULD_PROFILE`).  Batches are committed immediately if `Settings.INDEXING_JOB.SHOULD_COMMIT_BATCHES`, and not otherwise.  So, you can create a job to reindex a batch of PIDs by doing something like `IndexingJob.delay(priority: priority).perform_later(pid_list)`.
+#### Argo::LocalIndexingJob
+`Argo::LocalIndexingJob` is a subclass of `ActiveJob::Base`.  Its `perform` method just calls `Argo::Indexer.reindex_pid_list` (or its profiled counterpart, based on `Settings.INDEXING_JOB.SHOULD_PROFILE`).  Batches are committed immediately if `Settings.INDEXING_JOB.SHOULD_COMMIT_BATCHES`, and not otherwise.  So, you can create a job to reindex a batch of PIDs by doing something like `LocalIndexingJob.delay(priority: priority).perform_later(pid_list)`.
 
 #### Argo::BulkReindexer
 `Argo::BulkReindexer.new` expects a list of lists in the format returned by `PidGatherer.pid_lists_for_full_reindex` (described above).  Indeed, the static method `Argo::BulkReindexer.reindex_all` just creates an instance of its parent class using `pid_lists_for_full_reindex` from a new instance of `PidGatherer`.
 
 Once you've created a `BulkReindexer` instance with the desired `pid_lists`, you can have it index them by calling `queue_prioritized_pid_lists_for_reindexing`.  This method will:
 * iterate over `pid_lists` in order.
- * for each object type sub-list, it will break the list into chunks of `Settings.BULK_REINDEXER.BATCH_SIZE`, and create an `IndexingJob` for each chunk of PIDs.  It will also set the priority on each job, such that each batch has a priority corresponding to the object type contained in that batch (since the lists are already in priority order, it just uses the index of the parent list from `pid_lists`).  The delayed_job workers will then pick up the queued jobs in order of priority, and execute them.  Everything will get logged properly since the jobs are ultimately executing calls to `reindex_pid` (via a few intermediate layers).  Additionally, the delayed_job workers will log higher level info about their success/failure to execute.
+ * for each object type sub-list, it will break the list into chunks of `Settings.BULK_REINDEXER.BATCH_SIZE`, and create an `LocalIndexingJob` for each chunk of PIDs.  It will also set the priority on each job, such that each batch has a priority corresponding to the object type contained in that batch (since the lists are already in priority order, it just uses the index of the parent list from `pid_lists`).  The delayed_job workers will then pick up the queued jobs in order of priority, and execute them.  Everything will get logged properly since the jobs are ultimately executing calls to `reindex_pid` (via a few intermediate layers).  Additionally, the delayed_job workers will log higher level info about their success/failure to execute.
```

Comment:
Still exists.  The only change is that what was `IndexingJob` has been renamed to `LocalIndexingJob`.  But `LocalIndexingJob` still uses `Argo::Indexer`, which extends `Dor::IndexingService`, which implements `reindex_pid`:
https://github.com/sul-dlss/dor-services/blob/master/lib/dor/services/indexing_service.rb#L68

A larger question is whether the bulk reindexer that's documented in `INDEXING.md` (which was written for the argo 3 upgrade's reindexing) should live on as-is, get removed entirely, or be modified somehow.  I punted on that question for this PR, as I couldn't come up with an immediately obvious answer.  It still seems like that mechanism might be useful for a full reindex of DOR, though I'd be inclined to modify it so that it too calls out to dor_indexing_app to do the actual indexing work.  I'd be hesitant to do a full reindex of 1.5 million objects using this new bulk reindexing UI, since it does all the objects in one large batch (which should be fine for 10k or so objects, but maybe not for 100k or 500k).  Since the `BulkReindexer` mechanism documented here has batching, it seems like it'd be more sensible to just swap in the remote reindexing call, or have that `BulkReindexer` mechanism use the `RemoteIndexingJob` and then just remove the `LocalIndexingJob` part.